pkg/util/cloudproviders/cloudfoundry/cccache_test.go

```diff
@@ -261,6 +262,8 @@ func TestCCCache_GetProcesses(t *testing.T) {
 func TestCCCache_RefreshCacheOnMiss_GetProcesses(t *testing.T) {
 	cc.refreshCacheOnMiss = true
 	cc.reset()
+	// mark the cccache as updated once to trigger the refresh behaviour
+	cc.lastUpdated = time.Now().Add(time.Second)
```

Comment:
We only ever call `cc.reset()` in this context ie 
```
    cc.refreshCacheOnMiss = true
	cc.reset()
	// mark the cccache as updated once to trigger the refresh behaviour
	cc.lastUpdated = time.Now().Add(time.Second)
    ...
```
Can we make this a utility function? Or perform it all in reset()?

Reply:
Makes sense, I think that would be nice!